libraries/testAutomation/README.md

```diff
@@ -12,7 +12,7 @@ pipenv install
 
 To run the test automation library, a command like below can be used:
 ```shell
-pipenv run app
+pipenv run app --source-version=ES_5.6
```

Comment:
Looks like this isn't required, it uses the default which is the same as what you've included here.  Lets add a new section to the documentation about this.